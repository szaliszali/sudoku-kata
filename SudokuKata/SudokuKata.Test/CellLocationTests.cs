namespace SudokuKata.Test;

internal class CellLocationTests
{
    [Test]
    public void BlockIndex()
    {
        Approvals.VerifyAll(Enumerable.Range(0, 81).Select(i => new CellLocation(i / 9, i % 9).BlockIndex()), "");
    }
}
