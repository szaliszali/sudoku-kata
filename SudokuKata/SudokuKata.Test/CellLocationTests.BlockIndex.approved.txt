[0] = 0
[1] = 0
[2] = 0
[3] = 1
[4] = 1
[5] = 1
[6] = 2
[7] = 2
[8] = 2
[9] = 0
[10] = 0
[11] = 0
[12] = 1
[13] = 1
[14] = 1
[15] = 2
[16] = 2
[17] = 2
[18] = 0
[19] = 0
[20] = 0
[21] = 1
[22] = 1
[23] = 1
[24] = 2
[25] = 2
[26] = 2
[27] = 3
[28] = 3
[29] = 3
[30] = 4
[31] = 4
[32] = 4
[33] = 5
[34] = 5
[35] = 5
[36] = 3
[37] = 3
[38] = 3
[39] = 4
[40] = 4
[41] = 4
[42] = 5
[43] = 5
[44] = 5
[45] = 3
[46] = 3
[47] = 3
[48] = 4
[49] = 4
[50] = 4
[51] = 5
[52] = 5
[53] = 5
[54] = 6
[55] = 6
[56] = 6
[57] = 7
[58] = 7
[59] = 7
[60] = 8
[61] = 8
[62] = 8
[63] = 6
[64] = 6
[65] = 6
[66] = 7
[67] = 7
[68] = 7
[69] = 8
[70] = 8
[71] = 8
[72] = 6
[73] = 6
[74] = 6
[75] = 7
[76] = 7
[77] = 7
[78] = 8
[79] = 8
[80] = 8
